### PR TITLE
Release fixes

### PR DESCRIPTION
**Pull request by heckj**
- updating to reference 14.04.4 for packer - 14.04.3 is no longer available there
- updating to grab monorail-undionly.kpxe since that was set as latest code defaults

### CODE REVIEW

**Comment by heckj** [comment]
@RackHD/corecommitters for review please

**Comment by benbp** [comment]
👍

File: packer/ansible/roles/images/tasks/main.yml [hunk 1]
```diff
@@ -28,6 +28,7 @@
    - monorail-undionly.kpxe
    - monorail-efi32-snponly.efi
    - monorail-efi64-snponly.efi
+   - monorail-undionly.kpxe
```
Comment (anhou) [inline packer/ansible/roles/images/tasks/main.yml]: line 28 has include this file :)

**Comment by anhou** [comment]
:+1: with minor comments

**Comment by heckj** [comment]
build failure from known existing issue: https://github.com/RackHD/RackHD/issues/171 - going ahead with merges since earlier runs passed, and the build failure is a separate (intermittent) known issue and previous builds here were clean.

**Pull request merged**